src/main/java/ch/wisv/payments/admin/products/ProductServiceImpl.java

```diff
@@ -130,13 +142,32 @@ public boolean isProductAvailable(Integer productId) {
     }
 
     @Override
-    public void deleteProduct(int productId) {
-        List<Order> orders = orderService.getOrdersByProductId(productId);
+    public ProductGroup getProductGroupById(Integer productGroupId) {
+        return productGroupRepository.findOne(productGroupId);
+    }
 
-        if (orders.size() > 0) {
-            throw new ProductInUseException("Products are already ordered");
+    @Override
+    public void editProductGroup(ProductGroupRequest productGroupRequest) {
+        if (productGroupRequest.getId() != 0) {
+            Committee committee = committeeRepository.findOne(productGroupRequest.getCommitteeId());
+            ProductGroup productGroup = productGroupRepository.findOne(productGroupRequest.getId());
+
+            productGroup.setCommittee(committee);
+            productGroup.setName(productGroupRequest.getName());
+            productGroup.setDescription(productGroupRequest.getDescription());
+            productGroup.setGroupLimit(productGroupRequest.getGroupLimit());
+
+            productGroupRepository.saveAndFlush(productGroup);
+        }
+    }
+
+    @Override
+    public void deleteProductGroup(int productGroupId) {
+        ProductGroup productGroup = productGroupRepository.findOne(productGroupId);
+        if (!productGroup.getProducts().isEmpty()) {
+            throw new ProductGroupInUseException("Product group must be empty");
```

Comment:
We can also add the removal of tickets from a group as a hoothub issue, since this is not very high prioroty. It is also not relevant to editing the product group, it would make more sense to have it in a different page (`/products/group/editproducts`or something similar).